crates/bevy_render/src/texture/image_texture_loader.rs

```diff
@@ -65,7 +65,7 @@ impl AssetLoader for ImageTextureLoader {
                     data = i.into_raw();
                 }
                 image::DynamicImage::ImageRgb8(i) => {
-                    let i = image::DynamicImage::ImageRgb8(i).into_rgba();
+                    let i = image::DynamicImage::ImageRgb8(i).into_rgba8();
```

Comment:
Maybe unrelated?

Reply:
Yes, `into_rgba` was deprecated in a dependency, so CI failed.